pkg/controller/download_controller.go

```diff
@@ -38,8 +38,13 @@ import (
 	"k8s.io/client-go/tools/leaderelection"
 	"k8s.io/client-go/tools/leaderelection/resourcelock"
 	"k8s.io/utils/clock"
+	"time"
 )
 
+const MAX_RETRY = 5
```

Comment:
why not put these constants to pkg/utils/constants.go?

Reply:
Move to pkg/utils/constants.go.